corehq/apps/export/models/new.py

```diff
@@ -2470,6 +2477,48 @@ def user_to_be_notified(cls, domain, user):
             )
         )
 
+
+class DataFile(models.Model):
+    domain = models.CharField(max_length=126, db_index=True)
+    filename = models.CharField(max_length=255)
+    description = models.CharField(max_length=255)
+    content_type = models.CharField(max_length=255)
+    blob_id = models.CharField(max_length=255)
+    content_length = models.IntegerField(null=True)
+
+    class Meta(object):
+        app_label = 'export'
+
+    def get_blob(self):
+        db = get_blob_db()
+        try:
+            blob = db.get(self.blob_id)
+        except (KeyError, NotFound) as err:
+            raise self.DoesNotExist(str(err))
```

Comment:
might be better to raise a different exception otherwise it might get confused with when a `DataFile` record itself doesn't exist